tools/travis-setup-db.sh

```diff
@@ -34,6 +34,9 @@ CASSANDRA_VERSION=${CASSANDRA_VERSION:-3.9}
 # Default ElasticSearch version
 ELASTICSEARCH_VERSION=${ELASTICSEARCH_VERSION:-5.6.9}
 
+MYSQL_VERSION=${MYSQL_VERSION:-8.0.20}
+
```

Comment:
I've just noticed how this is duplicated between `travis-pull-db.sh` and `travis-setup-db.sh` would be cool if it could be simplified, but we can live without it anyway 🤷